README.md

```diff
@@ -11,7 +11,9 @@
   <a href="https://github.com/readmeio/rdme"><img src="https://img.shields.io/github/workflow/status/readmeio/rdme/CI.svg?style=for-the-badge" alt="Build status"></a>
 </p>
 
-With `rdme`, you can manage your API definition (we support [OpenAPI](https://spec.openapis.org/oas/v3.1.0.html) or [Swagger](https://swagger.io/specification/v2/)) and sync it to your API reference docs on ReadMe. You can also access other parts of [ReadMe's RESTful API](https://docs.readme.com/reference), including syncing Markdown documentation with your ReadMe project and managing project versions.
+<!--alex ignore postman-postwoman-->
+
+With `rdme`, you can manage your API definition (we support [OpenAPI](https://spec.openapis.org/oas/v3.1.0.html), [Swagger](https://swagger.io/specification/v2/), and [Postman](https://www.postman.com/collection/)) and sync it to your API reference docs on ReadMe. You can also access other parts of [ReadMe's RESTful API](https://docs.readme.com/reference), including syncing Markdown documentation with your ReadMe project and managing project versions.
```

Comment:
Not sure if you want to link to the actual Postman collection spec here instead but that URL is https://schema.postman.com/ if you're curious.

Reply:
Ah yeah, I'll do that to be consistent

Reply:
I'm gonna keep the second instance as [the landing page for Postman Collections](https://www.postman.com/collection/) since that does a better job of explaining what they are.